packages/lib/src/spatial-navigation/components/virtualizedList/helpers/computeTranslation.ts

```diff
@@ -1,74 +1,57 @@
 import { ScrollBehavior } from '../VirtualizedList';
+import { getSizeInPxFromOneItemToAnother } from './getSizeInPxFromOneItemToAnother';
 
 const computeStickToStartTranslation = <T>({
   currentlyFocusedItemIndex,
   itemSizeInPx,
-  nbMaxOfItems,
-  numberOfItemsVisibleOnScreen,
   data,
-  listSizeInPx,
+  maxPossibleLeftAlignedIndex,
 }: {
   currentlyFocusedItemIndex: number;
   itemSizeInPx: number | ((item: T) => number);
-  nbMaxOfItems: number;
-  numberOfItemsVisibleOnScreen: number;
-  data?: T[];
-  listSizeInPx?: number;
+  data: T[];
+  maxPossibleLeftAlignedIndex: number;
 }) => {
   let scrollOffset: number;
-  if (typeof itemSizeInPx === 'number') {
-    const maxPossibleLeftAlignedIndex = Math.max(nbMaxOfItems - numberOfItemsVisibleOnScreen, 0);
-    const leftAlignedIndex = Math.min(currentlyFocusedItemIndex, maxPossibleLeftAlignedIndex);
-    scrollOffset = leftAlignedIndex * itemSizeInPx;
-  } else if (data && listSizeInPx) {
-    const maxPossibleLeftAlignedIndex = getLastLeftItemIndex<T>(data, itemSizeInPx, listSizeInPx);
-    currentlyFocusedItemIndex < maxPossibleLeftAlignedIndex
-      ? (scrollOffset = data
-          .slice(0, currentlyFocusedItemIndex)
-          .reduce((acc, item) => acc + itemSizeInPx(item), 0))
-      : (scrollOffset = data
-          .slice(0, maxPossibleLeftAlignedIndex)
-          .reduce((acc, item) => acc + itemSizeInPx(item), 0));
-  } else {
-    throw new Error('itemSizeInPx is a function but data or list size is not provided');
-  }
+  currentlyFocusedItemIndex < maxPossibleLeftAlignedIndex
```

Comment:
weird to have a ternary that does an affectation instead of an affectation that does a ternary 😁

Reply:
Done :)